mbq/metrics/utils.py

```diff
@@ -1,8 +1,14 @@
-def tag_dict_to_list(tags):
-    return sorted([
-        ':'.join([str(k), str(v)]) if v else k
-        for k, v in tags.items()
-    ])
+def tags_as_list(tags):
+    if tags is None:
+        return []
+    if isinstance(tags, (list, tuple)):
```

Comment:
Used `(list, tuple)` instead of Iterable checking here because I want this function to fail if someone passes in a single string. Open to alternative implementations of that and/or arguments for why we should be okay with a user passing in a single string.